config/prod/htan-dcc-tma-tnp.yaml

```diff
@@ -21,6 +21,7 @@ parameters:
     - "arn:aws:iam::426577889437:user/clarisse.lau"
     - "arn:aws:iam::696725769253:user/creason"
     - "arn:aws:iam::289701539771:user/dagutman"
+    - "arn:iam::292075781285:user/jmuhlich"
```

Comment:
shouldn't it be `arn:aws:iam::292...`?

Reply:
yes! sorry I copied this from slack and it changed some of the arn into a emoji for some reason.